modules/savanna/modules/workgroup-workspace/pages/workgroups/workspace-logs.adoc

```diff
@@ -0,0 +1,43 @@
+= Log
+:experimental:
+
+Log provides full-text search capabilities for workspace logs (including historical logs, with a default retrievable period of 30 days) and allows filtering of logs based on time, nodes, and components. By utilizing log, teams can not only quickly locate and resolve issues but also gain in-depth insights into system performance, user behavior, and operational auditing, ultimately enhancing system reliability and performance.
+
+[TIP]
+====
+The Log feature is only available to enterprise-level customers. If you need to use this feature, please contact support@tigergraph.com to enable it for your organization.
+====
+
+== How to Enable Log
+
+.Enable log when creating a workgroup:
```

Comment:
```suggestion
```
Several writers in the Cloud/Tools team seem to be using this "dot" syntax. This is not its intended use in Asciidoc. Please do not use it this way. It is should only be used as the caption for a figure, table, or code block.  Captions are not sentences, just headings.
https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#more-delimited-blocks

Looking at your content (words) and intended formatting, this sentence is redundant and should just be dropped.